### ISSUE TITLE
making nvda accessible in lotus notes application

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4401]
**Reported by rinki.gadodia10 on 2014-08-19 14:14**
I am working in a bank where we use lotus notes as mail client. I don't know the version of it. I am unable to use nvda to access it. can nvda be used with this client. please guide
**Blocking #4429**

**Comment by geoffshang** [comment]
IBM Notes (formerly Lotus Notes) has two interfaces, a desktop application and a web interface.  The reporter is talking about the web interface.

NVDA does indeed have some issues with the web interface, and these vary from browser to browser.

As the reporter was attempting to use it with Internet Explorer 6, which I doubt is fully supported by the web interface let alone NVDA, and as the report lacks any real specifics, I suggest we close this ticket in preference for something more modern and specific.

**Comment by derekriemer** [comment]
Should this still be open as IE6 is no longer supported?

**Comment by jcsteh** [comment]
Closing as per last two comments.
